sensor_msgs/LaserScan laser_scan
---
geometry_msgs/PoseStamped seam_pose
